foxglove/cmd/config.go

```diff
@@ -0,0 +1,35 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func executeConfig() error {
+	fmt.Printf("Enter an API key (will be written to %s)\n", viper.ConfigFileUsed())
+	var bearerToken string
+	fmt.Scanln(&bearerToken)
+	viper.Set("bearer_token", bearerToken)
+	err := viper.WriteConfigAs(viper.ConfigFileUsed())
+	if err != nil {
+		return fmt.Errorf("Failed to write config: %w", err)
+	}
+	return nil
+}
+
+func newConfigCommand() *cobra.Command {
+	configCmd := &cobra.Command{
+		Use:   "config",
```

Comment:
Is "config" the right name for this command? I'm trying to think of other examples (git is what comes to mind), but this implies that I'll manage some more involved configuration, rather than just adding a secret. I guess I'm _configuring_ the CLI to use this secret, but that phrasing seems indirect.

I'm not sure what's better, though... "auth create"? "auth secrets add"?